package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "WTraining",
+  "version": "0.1.0",
+  "description": "WTraining",
+  "engines": {
+    "node": "10.14.1",
```

Comment:
The Node version is deprecated, but in order to follow this training, we will use it anyway.

Reply:
I will update it